Content.Server/Devour/DevourSystem.cs

```diff
@@ -32,6 +44,9 @@ private void OnDoAfter(EntityUid uid, DevourerComponent component, DevourDoAfter
             if (component.ShouldStoreDevoured && args.Args.Target is not null)
             {
                 ContainerSystem.Insert(args.Args.Target.Value, component.Stomach);
+
+                //Stops the bleeding of the devoured target so the dragon doesn't leave a trail of random blood puddles behind
+                _bloodstreamSystem.TryModifyBleedAmount(args.Args.Target.Value, -100);
             }
```

Comment:
This should be fixed on the bloodstream end and not worked around externally.

Reply:
I'm not 100% sure I understood what you meant, does the event I made come close to it?